rosbag2_to_video/api.py

```diff
@@ -30,25 +30,50 @@ def get_stamp_from_image_msg(image_msg):
 
 def add_arguments_to_parser(argparser):
     argparser.add_argument(
-        'bagfile', help='Path to the bag (currently only supports sensor_msgs/msg/Image types)')
-    argparser.add_argument('-t','--topic', required=True, help='Name of the image topic')
-    argparser.add_argument('-o','--output', required=True, help='Output filename (currently only supports MP4)')
+        'bagfile',
+        help='Path to the bag'
+    )
+    argparser.add_argument(
+        '-t',
+        '--topic',
+        required=True,
+        help='Name of the image topic (currently only supports sensor_msgs/msg/Image or sensor_msgs/msg/CompressedImage types)' # noqa E501
+    )
+    argparser.add_argument(
+        '-o',
+        '--output',
+        required=True,
+        help='Output filename (currently only supports MP4)'
+    )
     argparser.add_argument('--fps', type=int, required=True, help='Output frames per second')
+    argparser.add_argument('--storage-id', type=str, default="sqlite3", help='Rosbag2 storage id')
+    argparser.add_argument('--codec', type=str, default="avc1", help='Video codec')
```

Comment:
could you check that the provided argument has 4 characters?
it would also be nice to have an option to list supported codecs, but that's out of the scope of this PR